classes/antpool.php

```diff
@@ -38,12 +38,15 @@ function get($type) {
 		$nonce = time();
 		$hmac_message = $this->username.$this->api_key.$nonce;
 		$hmac = strtoupper(hash_hmac('sha256', $hmac_message, $this->api_secret, false));
+		// Set Coin Type: BTC, LTC, ETH, ZEC
+		$coin = '';
```

Comment:
$coin is not a global variable so it doesnt work.
 I think it is better if we change
function get($type) {
to
function get($type, $currency) {
so you are able to request more than one currency with one instance of the antpool api.